### ISSUE TITLE
some question for dataparallel

### ISSUE THREAD

**Issue by WFLiu0327** [IIGROUP/MANIQA#33]
when i try to train this model on double GPUs, an error occurred, suggesting that in models.maniqa.py in line 117 x = torch.cat((x6, x7, x8, x9), dim=2) appeared some data in CUDA0, some data in cuda1, please ask how this problem should be solved? I don't see some suitable solution on the internet.

**Comment by TianheWu** [comment]
Hi, I know this question.
This reason is the problem of implementation. It just support one GPU to training.

**Comment by TianheWu** [comment]
One solution is to return the output of each layer directly, without using the SaveOut module
